dmaws/utils.py

```diff
@@ -79,6 +79,9 @@ def __init__(self, mapping, variables, **kwargs):
         self._cache = {}
         self._variables = merge_dicts(variables, kwargs)
 
+    def items(self):
```

Comment:
This will be a merge conflict with https://github.com/alphagov/digitalmarketplace-aws/pull/27/files#diff-82cedc58a55843ad9169fc5b6ea91af6R82 if both land, so we should probably chose whichever looks better and keep it